### PR TITLE
dreamshaper-xl-v2-turbo

### PR DESCRIPTION
**Pull request by paulcjh**

### CODE REVIEW

**Comment by paulcjh** [comment]
this is really good

**Comment by paulcjh** [comment]
Thanks

**Comment by paulcjh** [comment]
no, thank you!

File: examples/text-to-image/dreamshaper-xl-v2-turbo/pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+runtime:
+  container_commands:
+    - apt-get update
+    - apt-get install -y git
```
Comment (rossgray) [inline examples/text-to-image/dreamshaper-xl-v2-turbo/pipeline.yaml:4]: not sure you strictly need these lines (might speed up the build without them) but up to you

File: examples/text-to-image/dreamshaper-xl-v2-turbo/new_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    def load(self) -> None:
+        # Perform any operations needed to load your model here
+        print("Loading model...", flush=True)
+
+        self.pipe = AutoPipelineForText2Image.from_pretrained(
+            "lykon/dreamshaper-xl-v2-turbo", torch_dtype=torch.float16, variant="fp16"
+        )
+        # self.pipe.scheduler = DPMSolverMultistepScheduler.from_config(
+        #     pipe.scheduler.config
+        # )
```
Comment (rossgray) [inline examples/text-to-image/dreamshaper-xl-v2-turbo/new_pipeline.py:67]:
```suggestion
```

**Review by rossgray — APPROVED** [review]
some very minor comments, but looks good

**Pull request merged**